src/nebula/Client.ts

```diff
@@ -0,0 +1,212 @@
+/**
+ * Created by Wu Jian Ping on - 2021/06/11.
+ */
+
+import _ from 'lodash'
+import Connection from './Connection'
+import { ClientOption, Task, Endpoint } from './types'
+import NebulaError from './NebulaError'
+import { EventEmitter } from 'events'
+
+let AsyncResource
+let executionAsyncId
+let isSupported = false
+
+try {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const asyncHooks = require('async_hooks')
```

Comment:
I'm curious about why not just import `import async_hooks` directly, is it possible that the module can't be imported?

Reply:
BTW, I find you use `import` other place:  https://github.com/vesoft-inc/nebula-node/blob/5d71bfd743ae512d2dc490b47897022cce5d6d4b/src/nebula/types.ts

Reply:
`async_hooks`, added in nodejs v8.1.0, this module can be run on `nodejs` below ` v8.1.0`, allthough I have specifed v10.0.0 in `package.json`. we use `require` for detecting the target node compatibility. If it not support, our module can also working fine.

Reply:
Directory Layout:
.
├── /dist/                                    # Build results for publishing (this folder generated after `npm run build`, its for publish)
├── /specs/                                 # Misc knowledge and docs
├── /src/                                     # Source Dir 
├── /tests/                                  # Unit test case, if you have nebula server, Put your test case in this folder for testing
├── /tools/                                 # Tiny build system
└── ....                                         # Supporting files

Build process:
![image](https://user-images.githubusercontent.com/14068989/144972293-a00bc533-7c96-46c8-9eb6-df5d8d5c718a.png)